app/static/js/admin/events/create.js

```diff
@@ -220,6 +220,16 @@ $(function() {
         $('.display-image').addClass('hidden');
     });
 
+    /*When image modal opens, request images*/
+    $(document).on('click', 'a[href="#show-modal"][data-modal="image"]', function(e) {
+        e.preventDefault();
+        $('#images-ajax-loadpoint').load("/admin/media/view", function(response, status){
+            if (status == "error"){
+                alert("Sorry, there was an error loading the images.");
```

Comment:
Can we display the specific error if one exists?  In general `alert()` is not a good option most of the time. I'd rather see an error message on the screen, and even better would be if the error message could be the same one that is sent from Flask